pkg/proto/v0/settings.proto

```diff
@@ -90,4 +90,5 @@ message ListOption {
     int64 intValue = 2;
   }
   bool selected = 3;
+  string displayValue = 4;
```

Comment:
Let's stick to the [language style guide](https://developers.google.com/protocol-buffers/docs/style#message-and-field-names) and use `underscode_separated_names`. I would also have to update it on the accounts service.

Reply:
done :-)

Reply:
but in the other PR. will merge it into this one eventually